src/lib.rs

```diff
@@ -336,6 +369,32 @@ impl<'a> FileMmapOptions<'a> {
         self.protection = Some(protection);
         self
     }
+    
+    /// Set the target address hint.
+    ///
+    /// Insuficcient alignment will cause the mapping to fail on windows.
```

Comment:
I think this would be more clear as 'The address must be aligned to the system pagesize.'